challenges/2022-07-12-node-depths/Solution.java

```diff
@@ -46,7 +46,31 @@
 public class Solution {
     static int nodeDepths(BinaryTree root) {
 	// insert your solution here
-	return -1;
```

Comment:
Looks like you have your solution in two places. In this file, replace lines 49-73 with `return -1` and push up your changes.